python/ray/scripts/scripts.py

```diff
@@ -697,6 +698,47 @@ def stack():
     subprocess.call(COMMAND, shell=True)
 
 
+@cli.command()
+@click.option(
+    "--redis-address",
+    required=False,
+    type=str,
+    help="Override the redis address to connect to.")
+def timeline(redis_address):
+    if not redis_address:
+        import psutil
+        pids = psutil.pids()
+        redis_addresses = set()
+        for pid in pids:
+            try:
+                proc = psutil.Process(pid)
+                for arglist in proc.cmdline():
+                    for arg in arglist.split(" "):
+                        if arg.startswith("--redis-address="):
+                            addr = arg.split("=")[1]
+                            redis_addresses.add(addr)
+            except psutil.NoSuchProcess:
+                pass
+        if len(redis_addresses) > 1:
+            print("Found multiple active Ray instances: {}. ".format(
+                redis_addresses) +
+                  "Please specify the one to connect to with --redis-address.")
+            exit(1)
+        elif not redis_addresses:
+            print("Could not find any running Ray instance. "
+                  "Please specify the one to connect to with --redis-address.")
+            exit(1)
+        redis_address = redis_addresses.pop()
+    print("Connecting to Ray instance at {}.".format(redis_address))
```

Comment:
Can we use `logger.info` for all of these prints? That's what we do in other places in this file.